### PR TITLE
Replace Physics with Collidable in YAML prototypes

### PR DESCRIPTION
**Pull request by DrSmugleaf**

### CODE REVIEW

**Comment by metalgearsloth** [comment]
I think most of them just need deleting e.g. crate_base already has collidable.

File: Resources/Prototypes/Entities/Constructible/Power/cloning_machine.yml [hunk 1]
```diff
@@ -23,7 +23,7 @@
             - Impassable
             - MobImpassable
             - VaultImpassable
-    - type: Physics
+    - type: Collidable
```
Comment (Zumorica) [inline Resources/Prototypes/Entities/Constructible/Power/cloning_machine.yml]: bruh, you just duplicated the collidable component here...

File: Resources/Prototypes/Entities/Constructible/Power/lathe.yml [hunk 2]
```diff
@@ -14,7 +14,7 @@
       - Impassable
       - MobImpassable
       - VaultImpassable
-  - type: Physics
+  - type: Collidable
```
Comment (Zumorica) [inline Resources/Prototypes/Entities/Constructible/Power/lathe.yml]: here too

File: Resources/Prototypes/Entities/Constructible/Power/medical_scanner.yml [hunk 3]
```diff
@@ -25,7 +25,7 @@
       - Impassable
       - MobImpassable
       - VaultImpassable
-  - type: Physics
+  - type: Collidable
```
Comment (Zumorica) [inline Resources/Prototypes/Entities/Constructible/Power/medical_scanner.yml]: and here

File: Resources/Prototypes/Entities/Constructible/Power/reagent_dispenser_base.yml [hunk 4]
```diff
@@ -14,7 +14,7 @@
       - Impassable
       - MobImpassable
       - VaultImpassable
-  - type: Physics
+  - type: Collidable
```
Comment (Zumorica) [inline Resources/Prototypes/Entities/Constructible/Power/reagent_dispenser_base.yml]: here

File: Resources/Prototypes/Entities/Constructible/Power/vending_machines.yml [hunk 5]
```diff
@@ -3,7 +3,7 @@
   name: vending machine
   abstract: true
   components:
-  - type: Physics
+  - type: Collidable
```
Comment (Zumorica) [inline Resources/Prototypes/Entities/Constructible/Power/vending_machines.yml]: here too

File: Resources/Prototypes/Entities/Constructible/Storage/Closets/closet.yml [hunk 6]
```diff
@@ -33,7 +33,7 @@
       - MobImpassable
       - VaultImpassable
       - SmallImpassable
-  - type: Physics
+  - type: Collidable
```
Comment (Zumorica) [inline Resources/Prototypes/Entities/Constructible/Storage/Closets/closet.yml]: here

File: Resources/Prototypes/Entities/Constructible/Storage/StorageTanks/base_tank.yml [hunk 7]
```diff
@@ -21,7 +21,7 @@
       - MobImpassable
       - VaultImpassable
       - SmallImpassable
-  - type: Physics
+  - type: Collidable
```
Comment (Zumorica) [inline Resources/Prototypes/Entities/Constructible/Storage/StorageTanks/base_tank.yml]: here

File: Resources/Prototypes/Entities/Constructible/Storage/crate_base.yml [hunk 8]
```diff
@@ -26,7 +26,7 @@
       - Opaque
       - MobImpassable
       - SmallImpassable
-  - type: Physics
+  - type: Collidable
```
Comment (Zumorica) [inline Resources/Prototypes/Entities/Constructible/Storage/crate_base.yml]: here

File: Resources/Prototypes/Entities/Constructible/Walls/mirror.yml [hunk 9]
```diff
@@ -12,7 +12,7 @@
       layer: [Clickable]
   - type: Clickable
   - type: InteractionOutline
-  - type: Physics
+  - type: Collidable
```
Comment (Zumorica) [inline Resources/Prototypes/Entities/Constructible/Walls/mirror.yml]: here

File: Resources/Prototypes/Entities/Mobs/NPCs/animals.yml [hunk 10]
```diff
@@ -26,7 +26,7 @@
       sprite: Mobs/Animals/monkey.rsi
 
   - type: Clickable
-  - type: Physics
+  - type: Collidable
```
Comment (Zumorica) [inline Resources/Prototypes/Entities/Mobs/NPCs/animals.yml]: here

File: Resources/Prototypes/Entities/Mobs/NPCs/animals.yml [hunk 11]
```diff
@@ -79,7 +79,7 @@
       state: crawling
       sprite: Mobs/Animals/gorilla.rsi
 
-  - type: Physics
+  - type: Collidable
```
Comment (Zumorica) [inline Resources/Prototypes/Entities/Mobs/NPCs/animals.yml]: here

File: Resources/Prototypes/Entities/Mobs/NPCs/animals.yml [hunk 12]
```diff
@@ -115,7 +115,7 @@
       state: chicken-0
       sprite: Mobs/Animals/chicken.rsi
 
-  - type: Physics
+  - type: Collidable
```
Comment (Zumorica) [inline Resources/Prototypes/Entities/Mobs/NPCs/animals.yml]: here

File: Resources/Prototypes/Entities/Mobs/NPCs/animals.yml [hunk 13]
```diff
@@ -154,7 +154,7 @@
       state: butterfly
       sprite: Mobs/Animals/butterfly.rsi
 
-  - type: Physics
+  - type: Collidable
```
Comment (Zumorica) [inline Resources/Prototypes/Entities/Mobs/NPCs/animals.yml]: here

File: Resources/Prototypes/Entities/Mobs/NPCs/animals.yml [hunk 14]
```diff
@@ -203,7 +203,7 @@
       state: bat
       sprite: Mobs/Animals/bat.rsi
 
-  - type: Physics
+  - type: Collidable
```
Comment (Zumorica) [inline Resources/Prototypes/Entities/Mobs/NPCs/animals.yml]: here

File: Resources/Prototypes/Entities/Mobs/NPCs/animals.yml [hunk 15]
```diff
@@ -243,7 +243,7 @@
       state: 0
       sprite: Mobs/Animals/bee.rsi
 
-  - type: Physics
+  - type: Collidable
```
Comment (Zumorica) [inline Resources/Prototypes/Entities/Mobs/NPCs/animals.yml]: here

File: Resources/Prototypes/Entities/Mobs/NPCs/animals.yml [hunk 16]
```diff
@@ -279,7 +279,7 @@
       state: goat
       sprite: Mobs/Animals/goat.rsi
 
-  - type: Physics
+  - type: Collidable
```
Comment (Zumorica) [inline Resources/Prototypes/Entities/Mobs/NPCs/animals.yml]: here

File: Resources/Prototypes/Entities/Mobs/NPCs/animals.yml [hunk 17]
```diff
@@ -316,7 +316,7 @@
       state: goose
       sprite: Mobs/Animals/goose.rsi
 
-  - type: Physics
+  - type: Collidable
```
Comment (Zumorica) [inline Resources/Prototypes/Entities/Mobs/NPCs/animals.yml]: here

File: Resources/Prototypes/Entities/Mobs/NPCs/animals.yml [hunk 18]
```diff
@@ -356,7 +356,7 @@
       state: parrot
       sprite: Mobs/Animals/parrot.rsi
 
-  - type: Physics
+  - type: Collidable
```
Comment (Zumorica) [inline Resources/Prototypes/Entities/Mobs/NPCs/animals.yml]: here

File: Resources/Prototypes/Entities/Mobs/NPCs/animals.yml [hunk 19]
```diff
@@ -392,7 +392,7 @@
       state: snake
       sprite: Mobs/Animals/snake.rsi
 
-  - type: Physics
+  - type: Collidable
```
Comment (Zumorica) [inline Resources/Prototypes/Entities/Mobs/NPCs/animals.yml]: here

File: Resources/Prototypes/Entities/Mobs/NPCs/animals.yml [hunk 20]
```diff
@@ -431,7 +431,7 @@
       state: tarantula
       sprite: Mobs/Animals/spider.rsi
 
-  - type: Physics
+  - type: Collidable
```
Comment (Zumorica) [inline Resources/Prototypes/Entities/Mobs/NPCs/animals.yml]: here

File: Resources/Prototypes/Entities/Mobs/NPCs/animals.yml [hunk 21]
```diff
@@ -467,7 +467,7 @@
       state: crab
       sprite: Mobs/Animals/crab.rsi
 
-  - type: Physics
+  - type: Collidable
```
Comment (Zumorica) [inline Resources/Prototypes/Entities/Mobs/NPCs/animals.yml]: here

File: Resources/Prototypes/Entities/Mobs/NPCs/animals.yml [hunk 22]
```diff
@@ -503,7 +503,7 @@
       state: penguin
       sprite: Mobs/Animals/penguin.rsi
 
-  - type: Physics
+  - type: Collidable
```
Comment (Zumorica) [inline Resources/Prototypes/Entities/Mobs/NPCs/animals.yml]: here

File: Resources/Prototypes/Entities/Mobs/NPCs/carp.yml [hunk 23]
```diff
@@ -21,7 +21,7 @@
       sprite: Mobs/Aliens/Carps/carp_space.rsi
 
   - type: Clickable
-  - type: Physics
+  - type: Collidable
```
Comment (Zumorica) [inline Resources/Prototypes/Entities/Mobs/NPCs/carp.yml]: here

File: Resources/Prototypes/Entities/Mobs/NPCs/carp.yml [hunk 24]
```diff
@@ -71,7 +71,7 @@
       state: magicarp
       sprite: Mobs/Aliens/Carps/carp_magic.rsi
 
-  - type: Physics
+  - type: Collidable
```
Comment (Zumorica) [inline Resources/Prototypes/Entities/Mobs/NPCs/carp.yml]: here

File: Resources/Prototypes/Entities/Mobs/NPCs/mimic.yml [hunk 25]
```diff
@@ -23,7 +23,7 @@
   - type: Icon
     sprite: Constructible/Power/VendingMachines/cola.rsi
     state: normal
-  - type: Physics
+  - type: Collidable
```
Comment (Zumorica) [inline Resources/Prototypes/Entities/Mobs/NPCs/mimic.yml]: here

File: Resources/Prototypes/Entities/Mobs/NPCs/pets.yml [hunk 26]
```diff
@@ -20,7 +20,7 @@
     baseSprintSpeed : 5
   - type: InteractionOutline
   - type: Clickable
-  - type: Physics
+  - type: Collidable
```
Comment (Zumorica) [inline Resources/Prototypes/Entities/Mobs/NPCs/pets.yml]: here

File: Resources/Prototypes/Entities/Mobs/NPCs/pets.yml [hunk 27]
```diff
@@ -71,7 +71,7 @@
     - map: ["enum.DamageStateVisualLayers.Base"]
       state: corgi
 
-  - type: Physics
+  - type: Collidable
```
Comment (Zumorica) [inline Resources/Prototypes/Entities/Mobs/NPCs/pets.yml]: here

File: Resources/Prototypes/Entities/Mobs/NPCs/pets.yml [hunk 28]
```diff
@@ -107,7 +107,7 @@
     - map: ["enum.DamageStateVisualLayers.Base"]
       state: ian
 
-  - type: Physics
+  - type: Collidable
```
Comment (Zumorica) [inline Resources/Prototypes/Entities/Mobs/NPCs/pets.yml]: here

File: Resources/Prototypes/Entities/Mobs/NPCs/pets.yml [hunk 29]
```diff
@@ -143,7 +143,7 @@
     - map: ["enum.DamageStateVisualLayers.Base"]
       state: cat
 
-  - type: Physics
+  - type: Collidable
```
Comment (Zumorica) [inline Resources/Prototypes/Entities/Mobs/NPCs/pets.yml]: here

File: Resources/Prototypes/Entities/Mobs/NPCs/pets.yml [hunk 30]
```diff
@@ -216,7 +216,7 @@
     - map: ["enum.DamageStateVisualLayers.Base"]
       state: sloth
 
-  - type: Physics
+  - type: Collidable
```
Comment (Zumorica) [inline Resources/Prototypes/Entities/Mobs/NPCs/pets.yml]: here

File: Resources/Prototypes/Entities/Mobs/NPCs/xeno.yml [hunk 31]
```diff
@@ -23,7 +23,7 @@
       sprite: Mobs/Aliens/Xenos/xeno_hunter.rsi
 
   - type: Clickable
-  - type: Physics
+  - type: Collidable
```
Comment (Zumorica) [inline Resources/Prototypes/Entities/Mobs/NPCs/xeno.yml]: here

File: Resources/Prototypes/Entities/Mobs/Player/observer.yml [hunk 32]
```diff
@@ -6,7 +6,7 @@
   description: Boo!
   components:
   - type: Mind
-  - type: Physics
+  - type: Collidable
```
Comment (Zumorica) [inline Resources/Prototypes/Entities/Mobs/Player/observer.yml]: here

File: Resources/Prototypes/Entities/Mobs/Species/human.yml [hunk 33]
```diff
@@ -116,7 +116,7 @@
     - map: [ "enum.Slots.MASK" ]
     - map: [ "enum.Slots.HEAD" ]
 
-  - type: Physics
+  - type: Collidable
```
Comment (Zumorica) [inline Resources/Prototypes/Entities/Mobs/Species/human.yml]: here

File: Resources/Prototypes/Entities/Mobs/Species/human.yml [hunk 34]
```diff
@@ -281,7 +281,7 @@
       - map: ["hand-left"]
       - map: ["hand-right"]
 
-    - type: Physics
+    - type: Collidable
```
Comment (Zumorica) [inline Resources/Prototypes/Entities/Mobs/Species/human.yml]: here

File: Resources/Prototypes/Entities/Objects/Power/powercells.yml [hunk 35]
```diff
@@ -103,7 +103,7 @@
     visuals:
     - type: PowerChargerVisualizer
   - type: Anchorable
-  - type: Physics
+  - type: Collidable
```
Comment (Zumorica) [inline Resources/Prototypes/Entities/Objects/Power/powercells.yml]: here

File: Resources/Prototypes/Entities/Objects/Power/powercells.yml [hunk 36]
```diff
@@ -132,7 +132,7 @@
...
```
Comment (Zumorica) [inline Resources/Prototypes/Entities/Objects/Power/powercells.yml]: here

File: Resources/Prototypes/Entities/Objects/Power/powercells.yml [hunk 37]
```diff
@@ -160,7 +160,7 @@
   - type: Appearance
     visuals:
     - type: PowerChargerVisualizer
-  - type: Physics
+  - type: Collidable
```
Comment (Zumorica) [inline Resources/Prototypes/Entities/Objects/Power/powercells.yml]: here

File: Resources/Prototypes/Entities/Objects/Specific/janitor.yml [hunk 38]
```diff
@@ -43,7 +43,7 @@
       - Opaque
       layer:
       - Opaque
-  - type: Physics
+  - type: Collidable
```
Comment (Zumorica) [inline Resources/Prototypes/Entities/Objects/Specific/janitor.yml]: here

File: Resources/Prototypes/Entities/Objects/Specific/janitor.yml [hunk 39]
```diff
@@ -73,7 +73,7 @@
       bounds: "-0.25,-0.25,0.25,0.25"
       mask:
       - Impassable
-  - type: Physics
+  - type: Collidable
```
Comment (Zumorica) [inline Resources/Prototypes/Entities/Objects/Specific/janitor.yml]: here

File: Resources/Prototypes/Entities/Objects/Specific/janitor.yml [hunk 40]
```diff
@@ -177,7 +177,7 @@
       layers:
         - state: chempuff
           map: [ "enum.VaporVisualLayers.Base" ]
-    - type: Physics
+    - type: Collidable
     - type: Collidable
```
Comment (Zumorica) [inline Resources/Prototypes/Entities/Objects/Specific/janitor.yml]: here

File: Resources/Prototypes/Entities/Objects/Weapons/Guns/Projectiles/projectiles.yml [hunk 41]
```diff
@@ -21,7 +21,7 @@
       mask:
       - Impassable
       - MobImpassable
-  - type: Physics
+  - type: Collidable
```
Comment (Zumorica) [inline Resources/Prototypes/Entities/Objects/Weapons/Guns/Projectiles/projectiles.yml]: here

File: Resources/Prototypes/Entities/Objects/Weapons/Guns/Projectiles/projectiles.yml [hunk 42]
```diff
@@ -104,7 +104,7 @@
       bounds: "-0.2,-0.2,0.2,0.2"
       layer: 32
       mask: 30
-  - type: Physics
+  - type: Collidable
```
Comment (Zumorica) [inline Resources/Prototypes/Entities/Objects/Weapons/Guns/Projectiles/projectiles.yml]: here

File: Resources/Prototypes/Entities/item_base.yml [hunk 43]
```diff
@@ -15,7 +15,7 @@
       bounds: "-0.25,-0.25,0.25,0.25"
       layer:
       - Clickable
-  - type: Physics
+  - type: Collidable
```
Comment (Zumorica) [inline Resources/Prototypes/Entities/item_base.yml]: and here

**Review by Zumorica — CHANGES_REQUESTED** [review]
wew lad

**Comment by DrSmugleaf** [comment]
> wew lad

Begone fiend

**Pull request merged**